tests/tests.rs

```diff
@@ -54,6 +54,10 @@ fn cmd_combine() {
     assert!(from_utf8(&output.stdout).unwrap().find("¡Si!").is_some());
 }
 
+static FIRST_LINE: &'static str = "[sound:empty_00060_828-00066_164.es.mp3],0:01:00.828,empty,\"<img src=\"\"empty_00063_496.jpg\"\" />\",¡Si! ¡Aang ha vuelto!,Yay! Yay! Aang\'s back!,,,¡Lo sabía!,I knew it!\n";
+
+static LAST_LINE: &'static str = "[sound:empty_00077_124-00083_379.es.mp3],0:01:17.124,empty,\"<img src=\"\"empty_00080_251.jpg\"\" />\",\"El no ha hecho nada, Sokka, fué un accidente.\",Aang didn\'t do anything.,harás que vengan a nosotros.,\"You\'re leading them straight to us, aren\'t you?\",,\n";
```

Comment:
Note that changing to your suggestion _without_ updating `cli_test_dir` will mean that we lose the test that verifies `--end-at` actually had the desired effect.